lua/plugins/filetype.lua

```diff
@@ -210,6 +210,9 @@ vis.ftdetect.filetypes = {
 	javascript = {
 		ext = { "%.cjs$", "%.js$", "%.jsfl$", "%.mjs$", "%.ts$" },
 	},
+  jq = {
+    ext = { "%.jq$" },
+  },
```

Comment:
The indentation is off here. Please try to match the surrounding style (using tabs in this case).

Reply:
Pushed a commit that fixes this indentation as my default is indeed spaces.